Find a functionally equivalent code:android.graphics.Matrix.preSkew:COMMENT
Method Modifier: public      
Comment:/**
 * Preconcats the matrix with the specified skew.
 * M' = M * K(kx, ky, px, py)
 */

@@ -1,3 +1,4 @@
 {
-    return native_preSkew(native_instance, kx, ky, px, py);
+    native_preSkew(native_instance, kx, ky, px, py);
+    return true;
 }

